plaso/parsers/winreg_plugins/bam.yaml

```diff
@@ -0,0 +1,36 @@
+name: bam
```

Comment:
It looks like you're only using the filetime definition here, and the other entries are unused - in the case, please use the definition in filetime.yaml instead.